File: Sep3_PresentationTier/Domains/Entity/DebitCard.cs
namespace Domains.Entity;

public class DebitCard
{
    public long CardId { get; set; }
    public long CardNumber { get; set; }
    public string? ExpiryDate { get; set; }
    public int Cvv { get; set; }

    public DebitCard(long cardNumber, string? expiryDate, int cvv)
    {
        CardNumber = cardNumber;
        ExpiryDate = expiryDate;
        Cvv = cvv;
    }
}

File: SEP3_AppTier_DataTier/Shared/src/entity/DebitCard.java
package entity;

public class DebitCard {
  private Long cardId;
  private Long cardNumber;
  private String expiryDate;
  private int cvv;

  public DebitCard(Long cardNumber, String expiryDate, int cvv) {
    this.cardNumber = cardNumber;
    this.expiryDate = expiryDate;
    this.cvv = cvv;
  }

  public Long getCardId() {
    return cardId;
  }

  public void setCardId(Long cardId) {
    this.cardId = cardId;
  }

  public Long getCardNumber() {
    return cardNumber;
  }

  public void setCardNumber(Long cardNumber) {
    this.cardNumber = cardNumber;
  }

  public String getExpiryDate() {
    return expiryDate;
  }

  public void setExpiryDate(String expiryDate) {
    this.expiryDate = expiryDate;
  }

  public int getCvv() {
    return cvv;
  }

  public void setCvv(int cvv) {
    this.cvv = cvv;
  }
}
